dvc/compare.py

```diff
@@ -153,7 +153,7 @@ def to_csv(self) -> str:
             writer.writerow(row)
         return buff.getvalue()
 
-    def to_parallel_coordinates(self, output_path, color_by):
+    def to_parallel_coordinates(self, output_path, color_by=None):
```

Comment:
@daavoo, please add type hints when you add any functions, no need to do it in this PR though, can be separate.